tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/catalog/AbstractNebulaCatalog.java

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package org.apache.flink.connector.nebula.catalog;
+
+/**
+ * AbstractNebulaCatalog is used to get nebula schema
+ */
+public interface AbstractNebulaCatalog {
+
+    boolean graphSpaceExists(String graphSpace);
+    boolean tagExists(String graphSpace, String tag);
+    boolean edgeExists(String graphSpace, String edge);
+
+    void listSpaces();
+    void getEdges(String graphSpace);
+    void getTags(String graphSpace);
+
+    void getTagProperties(String graphSpace, String tag);
```

Comment:
why return void?

Reply:
catalog is in another pr, the code is temporary